src/script/widgets/ClassificationPanel.js

```diff
@@ -0,0 +1,317 @@
+/**
+ * Copyright (c) 2008-2011 The Open Planning Project
+ *
+ * Published under the GPL license.
+ * See https://github.com/opengeo/gxp/raw/master/license.txt for the full text
+ * of the license.
+ */
+
+Ext.namespace("gxp");
+
```

Comment:
Please add at least some API docs, it doesn't have to be much, but something at least.